content/en/docs/ops/ambient/install/helm-installation/index.md

```diff
@@ -0,0 +1,164 @@
+  -f @manifests/charts/istio-control/istio-discovery/ambient-values.yaml@
+{{< /text >}}
+
+### Installing the ztunnel component
+
+The `ztunnel` chart installs the ztunnel DaemonSet, which is the node-proxy component of ambient.
+
+{{< text syntax=bash snip_id=install_ztunnel >}}
+$ helm install ztunnel istio/ztunnel -n istio-system
+{{< /text >}}
+
+## Configuration
+
+To view support configuration options and documentation, run:
+
+{{< text syntax=bash >}}
+$ helm show values istio/istiod
+{{< /text >}}
+
+## Verifying the Installation
+
+### Verifying the workload status
+
+After installing all the components, you can check the Helm deployment status with:
+
+{{< text syntax=bash snip_id=show_components >}}
+$ helm list -n istio-system
+{{< /text >}}
+
+You can check the status of the deployed pods with:
+
+{{< text syntax=bash snip_id=check_pods >}}
+$ kubectl get pods -n istio-system
+{{< /text >}}
+
+### Verifying with the Sample Application
+
+After installing ambient with Helm, you can follow
+[Deploy the sample application](/docs/ops/ambient/getting-started/#bookinfo)
+guide to deploy the sample application and ingress gateways, and then you can
+[add your application to ambient](/docs/ops/ambient/getting-started/#addtoambient).
+
+## Uninstall
+
+You can uninstall Istio and its components by uninstalling the charts
+installed above.
+
+1. List all the Istio charts installed in `istio-system` namespace:
+
+    {{< text syntax=bash >}}
+    $ helm ls -n istio-system
+    NAME       NAMESPACE    REVISION UPDATED         STATUS   CHART        APP VERSION
+    istio-base istio-system 1        ... ... ... ... deployed base-1.0.0   1.0.0
+    istiod     istio-system 1        ... ... ... ... deployed istiod-1.0.0 1.0.0
+    {{< /text >}}
+
+1. (Optional) Delete any Istio gateway chart installations:
+
+    {{< text syntax=bash >}}
+    $ helm delete istio-ingress -n istio-ingress
+    $ kubectl delete namespace istio-ingress
+    {{< /text >}}
+
+1. Delete Istio CNI chart:
+
+    {{< text syntax=bash snip_id=delete_cni >}}
+    $ helm delete istio-cni -n istio-system
+    {{< /text >}}
+
+1. Delete Istio ztunnel chart:
+
+    {{< text syntax=bash snip_id=delete_ztunnel >}}
+    $ helm delete ztunnel -n istio-system
+    {{< /text >}}
+
+1. Delete Istio discovery chart:
+
+    {{< text syntax=bash snip_id=delete_discovery >}}
+    $ helm delete istiod -n istio-system
+    {{< /text >}}
+
+1. Delete Istio base chart:
+
+    {{< tip >}}
+    By design, deleting a chart via Helm doesn't delete the installed Custom
+    Resource Definitions (CRDs) installed via the chart.
+    {{< /tip >}}
+
+    {{< text syntax=bash snip_id=delete_base >}}
+    $ helm delete istio-base -n istio-system
+    {{< /text >}}
+
+1. Delete the `istio-system` namespace:
+
+    {{< text syntax=bash snip_id=delete_system_namespace >}}
+    $ kubectl delete namespace istio-system
+    {{< /text >}}
+
+### (Optional) Deleting CRDs Installed by Istio
```

Comment:
i think this can be a bullet point in the previous uninstall section after "Delete Istio base chart"

Reply:
Do you mean a red warning message like https://istio.io/latest/docs/setup/getting-started/#install?

Reply:
just added a suggestion, you can check. It can be the next step after deleting the base chart, and before deleting the system namespace?

Reply:
The suggestion seems to be better organized, thanks!